### DISCUSSION TITLE
想试验下子集群间的负载均衡但一直未生效，请求一直还是只到其中liveshow_dynamic.A这个集群上，不知有什么原因？

### DISCUSSION THREAD

**Discussion by zhhOceanfly** [bfenetworks/bfe#1022] [Q&A (问题和回答)]
<img width="488" alt="image" src="https://user-images.githubusercontent.com/15883495/157354597-eb97a950-2121-4108-93a5-e3f63eae6b15.png">

**Comment by iyangsj** [comment]
@zhhOceanfly 可以排查以下情况：
1. 是否由于会话保持，请求被固定转发给A
2.是否由于B集群故障，请求跨子集群重试转发至A
